src/backend/app/projects/project_crud.py

```diff
@@ -119,26 +154,18 @@ async def create_tasks_from_geojson(
                 polygon["geometry"]["type"] = "Polygon"
                 polygon["geometry"]["coordinates"] = polygon["geometry"]["coordinates"][
                     0
-                ]
-
-            db_task = db_models.DbTask(
-                project_id=project_id,
-                outline=wkblib.dumps(shape(polygon["geometry"]), hex=True),
-                project_task_index=index + 1,
-            )
-            db.add(db_task)
-            log.debug(
-                "Created database task | "
-                f"Project ID {project_id} | "
-                f"Task index {index}"
-            )
-
-        # Commit all tasks and update project location in db
-        db.commit()
-
-        log.debug("COMPLETE: creating project boundary, based on task boundaries")
-
-        return True
+                ]                  
+            query = f""" INSERT INTO tasks (project_id,outline,project_task_index) VALUES ( '{project_id}', '{wkblib.dumps(shape(polygon["geometry"]), hex=True)}', '{index + 1}');"""
```

Comment:
Definitely don't use f-strings to insert values into the db. It probably opens us up to SQL injection attacks.

I'm sure there is probably a recommended way from the Databases docs.